.env.docker.example

```diff
@@ -0,0 +1,18 @@
+PORT=4321
```

Comment:
When I run the docker container, they need some environment variables. Unfortunately, when I run it using the .env, it's not working because of the single quote or double quote. I am not sure why it works in the other project. Any idea?

Reply:
After I rechecked it, the local docker is not working for solar. So, having something like this is normal as long as we set the env variables.